metadata_mapper/validate_mapping.py

```diff
@@ -1,78 +1,26 @@
-import boto3
 import json
-import logging
-import os
 import requests
 import sys
 import urllib3
 
-from datetime import datetime
-from pathlib import Path
-from typing import Type, Union
+from typing import Type
 
 import settings
 import utilities
 
-from mappers.validator import Validator, ValidationErrors
+from validator.validator import Validator
+from validator.validation_mode import ValidationMode
+from validator.validation_log import ValidationLogLevel
```

Comment:
Again, I think this can just be:

`from validator import validator, validation_mode, validation_log`